support/modules/single-file

```diff
@@ -10,6 +10,17 @@ dest_dir_file="$FILES"/files/dest_dir
 filename_file="$FILES"/files/filename
 permissions_file="$FILES"/files/permissions
 
+safe_copy() {
+    if [ -n "$3" ]; then
```

Comment:
:thinking: Is this the best way of checking nr args

Reply:
It's essentially the same as `$# -ge 3`, both are saying the same thing. But now that you point it out, it occurred to me that the `$#` style is more resilient if `set -u` is enabled, so I'll change it!